config.ini

```diff
@@ -26,7 +36,14 @@
     "second_stage_filter_cutoff" : "100.0e3",
     "second_stage_filter_transition" : "50.0e3",
     "third_stage_filter_cutoff" : "3.333e3",
-    "third_stage_filter_transition" : "0.833e3"
+    "third_stage_filter_transition" : "0.833e3",
+    "driver_to_rx_dsp_address" : "tcp://127.0.0.1:5000",
+    "radar_control_to_rx_dsp_address" : "tcp://127.0.0.1:5001",
+    "radar_control_to_driver_address" : "tcp://127.0.0.1:5002",
+    "rx_dsp_to_radar_control_ack_address" : "tcp://127.0.0.1:5003",
+    "rx_dsp_to_radar_control_timing_address" : "tcp://127.0.0.1:5004",
+    "experiment_to_control_socket" : "ipc:///tmp/feeds/1",
+    "data_to_experiment_socket" : "ipc:///tmp/feeds/2"
```

Comment:
Is acf processing happening in rx dsp or will it be its own separate block? This socket is for raw or fit data.